.secret.env.example

```diff
@@ -0,0 +1,6 @@
+PROVIDER_FACEBOOK_CLIENT_ID=437285735156519
+PROVIDER_FACEBOOK_CLIENT_SECRET=4e7db7b332555544c36ace89e209df73
+PROVIDER_TWITTER_CLIENT_ID=Y52pVLGPufTuooxltT2D2mK5E
+PROVIDER_TWITTER_CLIENT_SECRET=DZieOGVBSd9wSOuK7v88sEeLL8zyCiDBO4QBeHmmsUuDVCOt5Y
+PROVIDER_GOOGLE_CLIENT_ID=417471598206-j737cy0nezh5lab949ab428bsgzxipov.apps.googleusercontent.com
+PROVIDER_GOOGLE_CLIENT_SECRET=yk21KxJdr_fjIWPez72VSuVL
```

Comment:
@artstorm Ah! phew, thats ok then keep it like that. I think it is easier for a dev to replace something that looks like what they have.